src/sentry/snuba/metrics_layer/query.py

```diff
@@ -380,17 +376,6 @@ def fetch_namespace(metric: Metric) -> str:
     return namespaces.pop()
 
 
-def _resolve_metrics_entity(mri: str) -> EntityKey:
-    parsed_mri = parse_mri(mri)
-    if parsed_mri is None:
-        raise InvalidParams(f"'{mri}' is not a valid MRI")
-
-    if parsed_mri.namespace == "sessions":
-        return RELEASE_HEALTH_ENTITIES[parsed_mri.entity]
```

Comment:
Can you also remove the `RELEASE_HEALTH_ENTITIES` and `GENERIC_ENTITIES` dictionaries? They're not needed anymore.